function/pkg/operator/triggers_operator.go

```diff
@@ -4,22 +4,50 @@ import (
 	"context"
 
 	"github.com/kyma-incubator/hydroform/function/pkg/client"
+	"github.com/kyma-incubator/hydroform/function/pkg/resources/types"
 	"github.com/pkg/errors"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 )
 
 const message = "ownerID"
 
+type FnRef struct {
```

Comment:
Maybe we could make this type private, I don't think it's used anywhere outside this package?